.github/fabricbot.json

```diff
@@ -444,7 +444,57 @@
           }
         ]
       }
+    },
+    {
+      "taskType": "trigger",
+      "capabilityId": "IssueResponder",
+      "subCapability": "PullRequestResponder",
+      "version": "1.0",
+      "config": {
+        "conditions": {
+          "operator": "and",
+          "operands": [
+            {
+              "name": "prTargetsBranch",
+              "parameters": {
+                "branchName": "vs*"
+              }
+            },
+            {
+              "operator": "not",
+              "operands": [
+                {
+                  "name": "prModifiesFiles",
+                  "parameters": {
+                    "pathFilters": [
+                      "eng/Versions.props"
```

Comment:
We mentioned dropping this and just unconditionally saying "did you bump the version?" when we discussed this last (just writing it down so we don't forget)